src/server/redis_connection.h

```diff
@@ -90,6 +90,9 @@ class Connection {
   uint32_t GetPort() { return port_; }
   void SetListeningPort(int port) { listening_port_ = port; }
   int GetListeningPort() { return listening_port_; }
+  void SetAnnounceIP(std::string ip) { announce_ip_ = std::move(ip); }
```

Comment:
i think i forgot to add opportunity to kill client via  announce addr

Reply:
You can help to add this logic if you don't mind.

Reply:
done